bad-practice.js

```
@@ -0,0 +1,11 @@
+// This file is for testing CodeRabbit's code review
+const password = "123456";
+eval("console.log('This could be dangerous')");
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)